lib/data/repositories/v1/ticket_repository.dart

```diff
@@ -52,7 +52,7 @@ class TicketRepository {
         productName: result.productName,
         id: result.id,
         transactionType: TransactionType.ticketSwipe,
-        timeUsed: result.dateUsed,
+        timeUsed: result.dateUsed!,
```

Comment:
We discussed that a ticket should always have a date and therefore should be safe?

Reply:
Yes, I wanted you to add a comment in the code documenting this. I just realized that we are not using the Receipt.fromTicketDto. Use that one instead and add a comment in that method, why a ticketDto will always have a date

Reply:
i feel a comment explaining that this value is always present may be unnecessary. I think you can deduce it from the code, as it would not make sense to use the bang (!) if the value was optional. What do you think?

If we were to include a comment saying something like "this value is safe and always present and is caused by Chopper not being able to parse the Swagger specification correctly" we would need to do this everywhere we are doing anything similar

Reply:
From your answer alone, I think it would be prudent to leave a comment there, since your explanation just now was wrong. Chopper is parsing it correctly, but since we are in the context of receipts, this should only ever be called on responses where we have requested used tickets. Hence, why it is safe to access it. The same endpoint is capable of returning unused tickets, which is why the OpenApiSpec is as it is.